reframe/core/pipeline.py

```diff
@@ -1303,6 +1303,7 @@ def run(self):
         self.job.num_tasks = self.num_tasks
         self.job.num_tasks_per_node = self.num_tasks_per_node
         self.job.num_tasks_per_core = self.num_tasks_per_core
+        self.job.num_tasks_per_socket = self.num_tasks_per_socket
```

Comment:
Please do a separate bugfix PR for this.

Reply:
Moved this to #1837.